CotEditor/tr.lproj/DocumentInspectorView.strings

```diff
@@ -0,0 +1,141 @@
+
+/* Class = "NSTextFieldCell"; title = "Length"; ObjectID = "H0d-Eg-DcW"; */
+"H0d-Eg-DcW.title" = "Uzunluk";
+
+/* Class = "NSTextFieldCell"; title = "Character"; ObjectID = "H2Z-jU-biZ"; */
+"H2Z-jU-biZ.title" = "Karakter";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedIsNilPlaceholder = "-"; ObjectID = "IXM-Of-uQb"; */
+"IXM-Of-uQb.ibShadowedIsNilPlaceholder" = "-";
+
+/* Class = "NSTextFieldCell"; title = "57"; ObjectID = "KLW-KC-WGQ"; */
+"KLW-KC-WGQ.title" = "57";
+
+/* Class = "NSTextFieldCell"; title = "Permissions"; ObjectID = "N0Y-nO-k3L"; */
+"N0Y-nO-k3L.title" = "İzinler";
+
+/* Class = "NSTextFieldCell"; title = "nakamuxu"; ObjectID = "RR6-YZ-uWg"; */
+"RR6-YZ-uWg.title" = "nakamuxu";
+
+/* Class = "NSTextFieldCell"; title = "28"; ObjectID = "S9r-8W-Va2"; */
+"S9r-8W-Va2.title" = "28";
+
+/* Class = "NSTextFieldCell"; title = "LF"; ObjectID = "VRc-cm-IAI"; */
+"VRc-cm-IAI.title" = "LF";
+
+/* Class = "NSTextFieldCell"; title = "Column"; ObjectID = "Wwr-gI-HPc"; */
+"Wwr-gI-HPc.title" = "Sütun";
+
+/* Class = "NSTextFieldCell"; title = "Line Endings"; ObjectID = "XBS-xa-s3p"; */
+"XBS-xa-s3p.title" = "Satır Sonları";
+
+/* Class = "NSTextFieldCell"; title = "2014-03-19 00:00"; ObjectID = "XZn-I3-FkB"; */
+"XZn-I3-FkB.title" = "2014-03-19 00:00";
+
+/* Class = "NSTextFieldCell"; title = "8"; ObjectID = "aT9-Qs-HXT"; */
+"aT9-Qs-HXT.title" = "8";
+
+/* Class = "NSTextFieldCell"; title = "1024 (512)"; ObjectID = "bLv-xG-Hgy"; */
+"bLv-xG-Hgy.title" = "1024 (512)";
+
+/* Class = "NSTextFieldCell"; title = "Count"; ObjectID = "cbf-0f-9rd"; */
+"cbf-0f-9rd.title" = "Sayım";
+
+/* Class = "NSTextFieldCell"; title = "Characters"; ObjectID = "e1m-Uo-4UK"; */
+"e1m-Uo-4UK.title" = "Karakter";
+
+/* Class = "NSTextFieldCell"; title = "Unicode (UTF-8)"; ObjectID = "ebe-II-qO1"; */
+"ebe-II-qO1.title" = "Unicode (UTF-8)";
+
+/* Class = "NSTextFieldCell"; title = "Current Line"; ObjectID = "f9g-dQ-g3K"; */
+"f9g-dQ-g3K.title" = "Geçerli Satır";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedIsNilPlaceholder = "-"; ObjectID = "fEO-TD-8lt"; */
+"fEO-TD-8lt.ibShadowedIsNilPlaceholder" = "-";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedNoSelectionPlaceholder = "-"; ObjectID = "fEO-TD-8lt"; */
+"fEO-TD-8lt.ibShadowedNoSelectionPlaceholder" = "-";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedNotApplicablePlaceholder = "-"; ObjectID = "fEO-TD-8lt"; */
+"fEO-TD-8lt.ibShadowedNotApplicablePlaceholder" = "-";
+
+/* Class = "NSTextFieldCell"; title = "768 (256)"; ObjectID = "gzD-jh-yod"; */
+"gzD-jh-yod.title" = "768 (256)";
+
+/* Class = "NSTextFieldCell"; title = "Created"; ObjectID = "hgp-Cc-KHJ"; */
+"hgp-Cc-KHJ.title" = "Oluşturma";
+
+/* Class = "CocoaBindingsConnection"; ibShadowedIsNilPlaceholder = "-"; ObjectID = "igM-5I-vUh"; */
+"igM-5I-vUh.ibShadowedIsNilPlaceholder" = "-";
+
+/* Class = "NSTextFieldCell"; title = "Unicode"; ObjectID = "kna-yU-liB"; */
+"kna-yU-liB.title" = "Unicode";
```

Comment:
This field was changed from "Unicode" to "Code Points" in one of the latest commits 7a3676ada101c43e60375b106b3519844c7fdf4b. Could you suggest a new localized term for "Code Points"?

Reply:
Changed.